docs/src/univariate-list.md

```diff
@@ -0,0 +1,269 @@
+# [Univariate Distributions List](@id univariates-list)
```

Comment:
Is is not redundant with `univariates.md`? Plus it creates a hard to track distributions list that needs to be updated. The issue tracker might be more adapted

Reply:
Re redundancy:  
 - Think of this as an index page, with all the functionality and convenience use cases that index pages were created for.
 - Also think of this as a cross reference tool - not all distributions are known by the same name across all SME domains.

Happy to stand corrected on the cross referencing.

Yes this distributions list would need to be updated as distributions are added - more importantly reviews would be encouraged to identify synonyms that should be cross referenced to the incoming distribution.

I think the current issues are only in the search path of devs, not general users.